Content.Shared/Construction/Conditions/NoWindowsInTile.cs

```diff
@@ -11,9 +11,11 @@ public sealed class NoWindowsInTile : IConstructionCondition
     {
         public bool Condition(EntityUid user, EntityCoordinates location, Direction direction)
         {
-            var tagSystem = IoCManager.Resolve<IEntitySystemManager>().GetEntitySystem<TagSystem>();
+            var sysMan = IoCManager.Resolve<IEntitySystemManager>();
+            var tagSystem = sysMan.GetEntitySystem<TagSystem>();
+            var lookup = sysMan.GetEntitySystem<EntityLookupSystem>();
 
-            foreach (var entity in location.GetEntitiesInTile(LookupFlags.Approximate | LookupFlags.Static))
+            foreach (var entity in lookup.GetEntitiesIntersecting(location, LookupFlags.Approximate | LookupFlags.Static))
             {
```

Comment:
Doesn't this change the bounds being checked? I thought GetEntitiesInTile used an AABB to the tile (which also factors in grid.TileSize) whereas location only takes a single point.

Reply:
Can we use anchored tiles here? I don't know if all the collision tiles relevant for windows count as anchored.

Reply:
Thindows wouldn't get returned by this as they don't collide with the center.